features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/Resources.jsx

```diff
@@ -86,6 +86,8 @@ export default function Resources(props) {
                 setSelectedOperation({});
                 return data || openAPISpec.paths;
             case 'description':
+                updatedOperation[action] = value;
+                break;
```

Comment:
I think , If we do this [change](https://github.com/wso2/carbon-apimgt/pull/7505/files#r334918650) we could merge this back (revert).